index.html

```diff
@@ -179,17 +180,32 @@
         loadData(uri);
         document.getElementById('uriEntered').innerHTML = "You are displaying documents in " + uri.bold();
       }
-      else { //default
+      else {
         loadData('addr.json');
       }
 
+      $("input[name='insert']").click(function() {
```

Comment:
If you're already using jQuery, you should use [.serialize()](https://api.jquery.com/serialize/) for this.